x-pack/legacy/plugins/uptime/public/components/connected/alerts/alert_monitor_status.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { selectMonitorStatusAlert } from '../../../state/selectors';
+import { AlertMonitorStatusComponent } from '../../functional/alerts/alert_monitor_status';
+
+interface Props {
+  autocomplete: any;
```

Comment:
` autocomplete: DataPublicPluginSetup['autocomplete']`

Reply:
Fixed in https://github.com/elastic/kibana/pull/57919/commits/671b5ed284f0c39d4ea336a93ce2ed6c6376f9e6.